Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean

```diff
@@ -81,7 +81,8 @@ elliptic curve, weierstrass equation, j invariant
 
 -- porting note: replaced `map_one`, `map_bit0`, and `map_bit1` with `map_ofNat`
 local macro "map_simp" : tactic =>
-  `(tactic| simp only [map_ofNat, map_neg, map_add, map_sub, map_mul, map_pow])
+  `(tactic| simp only [Units.coe_map, Units.coe_map_inv, MonoidHom.coe_coe,
+      map_ofNat, map_neg, map_add, map_sub, map_mul, map_pow])
```

Comment:
If you added a bunch of lemmas to `map_simp` for `comp`, you might as well do the same for `id`. On the other hand I personally wouldn't add those lemmas to `map_simp` though, since they're only specific to `Units` or `MonoidHom`.

Reply:
Oh OK. Let me revert it to the previous state.